docs/superpowers/plans/2026-04-20-career-ops-saas-build.md

```
@@ -0,0 +1,2848 @@
+```
+
+Create `career-ops-web/vitest.setup.ts`:
+
+```typescript
+import '@testing-library/jest-dom'
+```
+
+- [ ] **Step 8: Add test script to package.json**
+
+In `package.json` scripts, add:
+```json
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)